Dockerfile

```
@@ -1,15 +1,46 @@
-RUN apt update && apt install curl git -y
+# Install system dependencies and clean up in single layer
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        curl \
+        git \
+    && apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
-COPY . .
+# Install helm-docs with checksum verification
+RUN curl -L -o /tmp/helm-docs.tar.gz \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)